libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts

```diff
@@ -190,5 +191,33 @@ describe('withUndoRedo', () => {
         expect(store.canRedo()).toBe(true);
       });
     }));
+
+    it('clears undo redo stack', fakeAsync(() => {
+      TestBed.runInInjectionContext(() => {
```

Comment:
I know that you did it as the other tests before, but could please do it without runinInjectionContext? So you setup the signalStore first (with providedIn root), and then you just say TestBed.inject(Store)?

Reply:
It's fixed. Thank you